src/lib/healthCheck/HealthCheckServer.js

```diff
@@ -0,0 +1,108 @@
+
```

Comment:
Do you not think that we should create a separate lib for this?

I did say that we should probably not add this to the common lib, but perhaps it would be better to add this to a new lib specifically for health checks so you don't have to copy pasta?

Reply:
Yeah I agree. Perhaps we can call it something like `mojaloop/central-services-health` ?

Reply:
And I can write it in typescript if you'd like :)